integration/hello_server/server_index.tsx

```diff
@@ -29,5 +29,5 @@ export async function serverMain(document: Document) {
       </body>
     </html>
   );
-  return jsxRender(document, doc, document);
+  return jsxRender(document, doc);
```

Comment:
ok, this is super confusing. `(document, doc)`. I know it was like this before but could we change it to something better? `(document, jsx)`? maybe?